packages/fluentui/react-northstar/src/components/Form/FormRadioGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import { compose } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import _FormFieldBase, { FormFieldBaseProps } from './utils/formFieldBase';
+import RadioGroup, { RadioGroupProps } from '../RadioGroup/RadioGroup';
+
+interface FormRadioGroupOwnProps extends RadioGroupProps {}
+type SelectedFormFieldCustomProps = Omit<
+  FormFieldBaseProps,
+  'control' | 'styles' | 'accessibility' | 'design' | 'variables'
+>;
+export interface FormRadioGroupProps extends SelectedFormFieldCustomProps, FormRadioGroupOwnProps {}
+export type FormRadioGroupStylesProps = never;
+
+export const formRadioGroupClassName = 'ui-forms';
```

Comment:
@layershifter For some very weird reason in the `FormRadioGroup.info.json` when it's being generated it's setting:

`"componentClassName": "ui-forms",`

Reply:
Yep, we have this issue in few places. Let's keep as it is for now